tethys_apps/migrations/0004_customsetting_default.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.1.8 on 2019-05-09 19:49
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('tethys_apps', '0003_python3_compatibility'),
```

Comment:
```suggestion
        ('tethys_apps', '0004_auto_20190506_1714'),
```

Reply:
It doesn't look like this change was made.

Reply:
oops. It has now.